portal/app/models.py

```diff
@@ -37,6 +37,10 @@ class DeviceType(Enum):
     CLOTHES_WASHER = 'CLOTHES_WASHER'
     AIR_CONDITIONER = 'AIR_CONDITIONER'
     STOVE_OVEN_EXHAUST = 'STOVE_OVEN_EXHAUST'
+    SONNEN = 'SONNEN'
+    EGAUGE = 'EGAUGE'
+    BLENDER = 'BLENDER'
```

Comment:
Granted I don't think we'll ever add an actual home blender (for food) to this list, it may be better to be more explicit, like `PICO_BLENDER` instead of just `BLENDER`.  Same thing for SONNEN, but to a lesser extent I guess